plans/academic_onramp.markdown

```diff
@@ -0,0 +1,122 @@
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+The challenge is how to improve the onramp for students both before they start Module 1 and the first few days of the module.
+
+* Why is this a “challenge”?
+
+This is a challenge because students come in with varying degrees of skill and aptitude, and to get them to all reach a baseline of 
+knowledge is difficult, epsecially with a sort of a "rolling" admissions policy when people commit to a start date at different times.
+On top of that, a number of them are moving to Denver, so the time that should be dedicated to pre-work is combined with a very high
+level of stress. However, there is not enough data to conclude if there is a correlation between moving to Denver for Turing and
+module 1 success.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+I don't know if it is something that can be "solved," but it is certainly something that can be improved on.
+
+* What happens if we fail to solve it / improve it?
+
+If we fail to solve it, we will continue to have issues with frustrated students, and confidence levels, potentially resulting in an 
+unacceptable number of students repeating module 1.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+The Module 1 experience is freshest in my mind and I spend a lot of time with the current first cohort, so I have a good insight into
+what they are going through, and an insight into their challenges and their successes.
+
+
+## The Past
+
+* What’s been done so far?
+
+So far, pre-work has been very hands off. It's been here's Pragmatic Studio, let us know if you have any questions. We are starting
+to get more organized with Q and A sessions.
+
+* Was it effective? How do you know?
+
+I would say that it has been at least somewhat effective. Turing has graduated four cohorts so far, with only, as far as I know, 
+three individuals unable to proceed past module 1.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+I do not know.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now? 
+
+I would like whoever is teaching Module 1 to be more active in the preparation of the incoming cohort. I'd like to schedule short 
+meetings each to gauge where they are and how they feel.
+
+* What will be the next three things you do?
+
+I would give 1507 access to Slack right now. I would write a series of diagnostics for 1507 so that we have a better idea of where
+the group stands coming into module one and not say after their first project. I would assign them to posses right now so they meet
+their posse-mates and their mentors.
```

Comment:
The tricky part about this is that the incoming cohort roster tends to change right up until day 1.